__tests__/app.test.js

```diff
@@ -182,6 +182,25 @@ describe("/api/articles", () => {
         });
       });
   });
+  test("GET 200: return articles filtered by topics", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.articles.length).toBe(12);
+        response.body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("GET 404: return error message if the topic was not found", () => {
+    return request(app)
+      .get("/api/articles?topic=banana")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Topic Not Found");
+      });
+  });
```

Comment:
I think one good test here would be if the query has a key you weren't expecting e.g. ?mystery=banana

it might not be that we want to throw an error here (maybe we will query by mystery in the future) but we might want to just ignore the query altogether and send all the articles back